src/components/chart/chart.js

```diff
@@ -114,6 +114,8 @@ class HighchartsBaseChart {
         }
         this.config.chart.events.load = (event) => {
             const currentChart = event.target;
+            // Disable the legend for single series charts
+            this.commonChartOptions.disableLegendForSingleSeries(currentChart);
```

Comment:
This one might be worth a quick chat at stand up. Given that we allow editors to display or hide a legend, it might be better to leave it to them to follow the guidelines rather than enforcing here - although we already do something similar for data labels in bar charts with more than a certain number of series.

Reply:
Agreed to commit this for now and raise a separate ticket if we don't want to enforce this.